src/lib.rs

```diff
@@ -80,7 +80,8 @@ pub struct Config {
     #[serde(default)]
     pub save_window_position: bool,
     pub window_position: Option<WindowPosition>,
-    options: Options,
+    pub url: String,
```

Comment:
this should be an `Option`
```suggestion
    pub url: Option<String>,
```